tools/enrollment_preprocessor.py

```diff
@@ -88,24 +89,24 @@ def get_user_decisions(database_users: dict[str, User], workbook: Workbook) -> d
                     conflicts_by_field[field].add(imported)
     # ask user for decision
     for field, conflicts in conflicts_by_field.items():
-        print(field.capitalize())
-        print("---------")
+        _stdout.write(f"{field.capitalize()}\n")
+        _stdout.write("---------\n")
         for imported in sorted(conflicts):
             existing = database_users[imported.email]
 
             if getattr(existing, field) == getattr(imported, field):
                 continue
 
-            print(f"existing: '{make_bold(getattr(existing, field))}' ({existing.full_name()})")
-            print(f"imported: '{make_bold(getattr(imported, field))}' ({imported.full_name()})")
+            _stdout.write(f"existing: '{make_bold(getattr(existing, field))}' ({existing.full_name()})\n")
+            _stdout.write(f"imported: '{make_bold(getattr(imported, field))}' ({imported.full_name()})\n")
 
             decision = ""
             while decision not in ("e", "i"):
                 decision = input("Which one should be used? (e/i):\n")
             choice = existing if decision == "e" else imported
 
             database_users[choice.email] = database_users[choice.email]._replace(**{field: getattr(choice, field)})
-        print()
+            _stdout.write("\n")
```

Comment:
semantincal change (by changed indentation) -- unintended?

Reply:
good catch, yeah this was not intended